packagehandlers/conanpackagehandler.go

```diff
@@ -0,0 +1,79 @@
+package packagehandlers
+
+import (
+	"fmt"
+	"github.com/jfrog/frogbot/v2/utils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"os"
+	"strings"
+)
+
+const (
+	conanFileTxt = "conanfile.txt"
+	conanFilePy  = "conanfile.py"
+)
+
+type ConanPackageHandler struct {
+	CommonPackageHandler
+}
+
+func (conan *ConanPackageHandler) UpdateDependency(vulnDetails *utils.VulnerabilityDetails) error {
+	if vulnDetails.IsDirectDependency {
+		return conan.updateDirectDependency(vulnDetails)
+	} else {
+		return &utils.ErrUnsupportedFix{
+			PackageName:  vulnDetails.ImpactedDependencyName,
+			FixedVersion: vulnDetails.SuggestedFixedVersion,
+			ErrorType:    utils.IndirectDependencyFixNotSupported,
+		}
+	}
+}
+
+func (conan *ConanPackageHandler) updateDirectDependency(vulnDetails *utils.VulnerabilityDetails) (err error) {
+	conanDescriptors, err := conan.CommonPackageHandler.GetAllDescriptorFilesFullPaths([]string{conanFileTxt, conanFilePy})
+	if err != nil {
+		err = fmt.Errorf("failed while searching for Conan descriptor files in project: %s", err.Error())
+		return
+	}
+	isAnyDescriptorFileChanged := false
+	for _, descriptor := range conanDescriptors {
+		var isFileChanged bool
+		isFileChanged, err = conan.updateConanFile(descriptor, vulnDetails)
+		if err != nil {
+			return
+		}
+		isAnyDescriptorFileChanged = isAnyDescriptorFileChanged || isFileChanged
+	}
+	if !isAnyDescriptorFileChanged {
+		err = fmt.Errorf("impacted package '%s' was not found or could not be fixed in all descriptor files", vulnDetails.ImpactedDependencyName)
+	} else {
+		conan.logNoInstallationMessage()
+	}
+	return
+}
+
+func (conan *ConanPackageHandler) updateConanFile(conanFile string, vulnDetails *utils.VulnerabilityDetails) (isFileChanged bool, err error) {
+	data, err := os.ReadFile(conanFile)
+	if err != nil {
+		return false, fmt.Errorf("an error occurred while attempting to read the requirements file '%s': %s\n", conanFile, err.Error())
+	}
+	currentFile := string(data)
+	fixedPackage := vulnDetails.ImpactedDependencyName + "/" + vulnDetails.SuggestedFixedVersion
+	impactedDependency := vulnDetails.ImpactedDependencyName + "/" + vulnDetails.ImpactedDependencyVersion
+	fixedFile := strings.Replace(currentFile, impactedDependency, strings.ToLower(fixedPackage), 1)
+
+	if fixedFile == currentFile {
+		log.Info(fmt.Sprintf("impacted dependency '%s' not found in descriptor '%s', moving to the next descriptor if exists...", impactedDependency, conanFile))
```

Comment:
```suggestion
		log.Debug(fmt.Sprintf("impacted dependency '%s' not found in descriptor '%s', moving to the next descriptor if exists...", impactedDependency, conanFile))
```
This message is not adding necessary informaion for someone who is not debugging and will load the logs. We want to notify about a descriptor that didnt have some dependency only in debug mode.